### PR TITLE
Added admin buttons allowing them to easily contact an entire faction or individual role

### LINKED ISSUE #30260: TODO: Admin buttons allowing to contact an entire strike team, just like the similar Cult button
noting it down so I don't forget it

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #30260

:cl:
* rscadd: Added admin buttons allowing them to easily contact an entire faction or individual role. Similarly to how they could already contact cultists as Nar-Sie (basically this feature has been expanded to all factions and roles). Admins can also set the name they use if they wish to use one different from the default.

### CODE REVIEW

File: code/datums/gamemode/factions/syndicate/rev.dm [hunk 1]
```diff
@@ -12,6 +12,8 @@
 	initroletype = /datum/role/revolutionary/leader
 	roletype = /datum/role/revolutionary
 	playlist = "nukesquad"
+	default_admin_voice = "The spirit of Revolution!"
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/syndicate/rev.dm]: Union Boss instead perhaps

File: code/datums/gamemode/factions/junglefever.dm [hunk 2]
```diff
@@ -5,6 +5,8 @@
 	name = MADMONKEY
 	ID = MADMONKEY
 	logo_state = "monkey-logo"
+	default_admin_voice = "big monkey in the sky"
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/junglefever.dm]: missed opportunity for Monkey King

File: code/datums/gamemode/role/rambler.dm [hunk 3]
```diff
@@ -5,6 +5,8 @@
 	special_role = RAMBLER
 	logo_state = "rambler-logo"
 	wikiroute = ROLE_MINOR
+	default_admin_voice = "Xavier"
```
Comment (Kurfursten) [inline code/datums/gamemode/role/rambler.dm]: Chief Master Guru is the name of Xavier's mentor, an Appachistani mystic

File: code/datums/gamemode/factions/junglefever.dm (hunk 2 above)
Comment (DeityLink) [reply code/datums/gamemode/factions/junglefever.dm]: agreed

File: code/datums/gamemode/role/rambler.dm (hunk 3 above)
Comment (DeityLink) [reply code/datums/gamemode/role/rambler.dm]: I've shamefully only watched like 2 episodes, thank you.

**Pull request merged**